domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -848,6 +852,10 @@ public void save(AsyncResult<SQLConnection> sqlConnection, String table, String
    * @param replyHandler  result, containing the id field for each inserted element of entities
    */
   public void saveBatch(String table, JsonArray entities, Handler<AsyncResult<ResultSet>> replyHandler) {
+    saveBatch(table, entities, DEFAULT_JSONB_FIELD_NAME, replyHandler);
```

Comment:
NOTE I started adding this to facilitate unit testing.  It's not directly related to DISTINCT ON